site/docs/Tutorial-BasicTypes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Basic Types
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/basic-types/
+next: /graphql-js/passing-arguments/
+---
+
+In most situations, all you need to specify the types for your API is the GraphQL schema language, taken as an argument to the `buildSchema` function.
+
+The GraphQL schema language supports the scalar types of `String`, `Int`, `Float`, `Boolean`, and `ID`, so you can use these directly in the schema you pass to `buildSchema`.
```

Comment:
You have to define enums with the 'enum' keyword so I think they belong in some other doc, after the "basic types" section. So let me punt on that til later.

I should confess, I have never used an enum type in graphql, I just pass strings :P